src/wp-includes/blocks/footnotes.php

```diff
@@ -68,17 +68,26 @@ function render_block_core_footnotes( $attributes, $content, $block ) {
  * @since 6.3.0
  */
 function register_block_core_footnotes() {
-	foreach ( array( 'post', 'page' ) as $post_type ) {
-		register_post_meta(
-			$post_type,
-			'footnotes',
-			array(
-				'show_in_rest'      => true,
-				'single'            => true,
-				'type'              => 'string',
-				'revisions_enabled' => true,
-			)
-		);
+	$post_types = get_post_types(
+		array(
+			'show_in_rest' => true,
+			'public'       => true,
+		)
+	);
+	foreach ( $post_types as $post_type ) {
+		// Only register the meta field if the post type supports the editor, custom fields, and revisions.
+		if ( post_type_supports( $post_type, 'editor' ) && post_type_supports( $post_type, 'custom-fields' ) && post_type_supports( $post_type, 'revisions' ) ) {
```

Comment:
I don't think custom fields is required: that is only used to determine whether the interface is rendered. Post meta exists regardless. 

https://github.com/WordPress/wordpress-develop/blob/bc4eb46b5577bae4fc48f0cebac0f0d037939db0/src/wp-admin/includes/meta-boxes.php#L1649-L1662

Reply:
cc @ellatrix 

I don't really know much about this code. Most of it is automatically added from the Gutenberg repo (all the blocks php file). Let's make sure we track any improvement in the Gutenberg repo.

Reply:
@youknowriad @ellatrix Upon further investigation, I suspect it's a REST API thing -- I'll get in touch with Tim.